### ISSUE TITLE
Support Laravel 6.0

### ISSUE THREAD

**Issue by leknoppix** [codezero-be/cookie#3]
The new version of Laravel is coming. I Have try to upgrade my project from 5.8 to 6.
I have this error:
```
bloglaravel>$ composer require codezero/cookie
Using version ^1.0 for codezero/cookie
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for laravel/framework (locked at v6.0.3, required as ^6.0) -> satisfiable by laravel/framework[v6.0.3].
    - codezero/cookie 1.0.0 requires codezero/encrypter ~1.0 -> satisfiable by codezero/encrypter[1.0.0].
    - codezero/cookie 1.0.1 requires codezero/encrypter ~1.0 -> satisfiable by codezero/encrypter[1.0.0].
    - codezero/cookie 1.0.2 requires codezero/encrypter ~1.0 -> satisfiable by codezero/encrypter[1.0.0].
    - Conclusion: don't install codezero/encrypter 1.0.0
    - Installation request for codezero/cookie ^1.0 -> satisfiable by codezero/cookie[1.0.0, 1.0.1, 1.0.2].
```

**Comment by leknoppix** [comment]
I think that this project is dead!

**Comment by ivanvermeyen** [comment]
Hi, Its been ages since I used this repo, but I will check if it is easy to make it compatible with Laravel 6. A pull request is also welcome 👍

**Comment by ivanvermeyen** [comment]
I updated the version constraints and I'm pretty sure it will work now.

I will need to change my test suite to PHPUnit, because PHPSpec is having issues...
Can't install both PHPSpec and `laravel/http` 6.x.

I'll try to tag a new version today.

**Comment by leknoppix** [comment]
Thanks for you response.
I'm looking your error in your test and I think that laravel6.0 request PHP >= 7.2.0.
Your test is only about php 5.6 or more, but not 7! No?

**Comment by ivanvermeyen** [comment]
Hi,

Yes, I encountered a few incompatibilities, so I decided to rewrite and simplify the code.
At the moment I tested the vanilla implementation as good as I can.

I'm not sure if I should test the Laravel implementation. All it kinda does is redirect calls to Laravel's `CookieJar`... That made me wonder, how do benefit from this package in a Laravel app? I mainly created this for vanilla PHP :)

I'm working out some doctype stuff that Scrutinizer is complaining about and then I think I'm ready to tag a new major version.

I already pushed all the recent changes. If you have any thoughts about it, feel free to tell me :)
